awx/main/management/commands/run_rsyslog_configurer.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from awx.main.dispatch import pg_bus_conn
+from awx.main.utils.external_logging import reconfigure_rsyslog
+
+logger = logging.getLogger('awx.main.rsyslog_configurer')
+
+
+class Command(BaseCommand):
+    """
+    Rsyslog Configurer
+    Runs as a management command and starts rsyslog configurer daemon. Daemon listens
+    for pg_notify then calls reconfigure_rsyslog
+    """
+
+    help = 'Launch the rsyslog_configurer daemon'
+
+    def handle(self, *arg, **options):
+        try:
+            with pg_bus_conn(new_connection=True) as conn:
+                conn.listen("rsyslog_configurer")
+                for e in conn.events(yield_timeouts=True):
+                    if e is not None:
+                        reconfigure_rsyslog()
```

Comment:
Before you call this, you should clear all local caches for settings. Otherwise we could face some race conditions.

Reply:
what is the local cache that ur referring to here?

Reply:
it's implemented now, `cache` and `settings._awx_conf_memoizedcache`